pkg/kn/commands/service/service_list_test.go

```diff
@@ -78,6 +91,55 @@ func TestServiceListDefaultOutput(t *testing.T) {
 	assert.Check(t, util.ContainsAll(output[2], "bar", "bar.default.example.com", "2"))
 }
 
+func TestServiceGetOneOutput(t *testing.T) {
+	service := createMockServiceWithParams("foo", "foo.default.example.com", 1)
+	serviceList := &v1alpha1.ServiceList{Items: []v1alpha1.Service{*service}}
+	action, output, err := fakeServiceList([]string{"service", "list", "foo"}, serviceList)
+	if err != nil {
+		t.Fatal(err)
+	}
+	if action == nil {
+		t.Errorf("No action")
+	} else if !action.Matches("list", "services") {
+		t.Errorf("Bad action %v", action)
+	}
+	assert.Check(t, util.ContainsAll(output[0], "NAME", "DOMAIN", "GENERATION", "AGE", "CONDITIONS", "READY", "REASON"))
```

Comment:
The DOMAIN field is being replaced in #247 with URL, one of these two PRs need to rebase and update based on which gets merged first. 

This PR is around since a while, probably we can consider updating #247 once this is in.
WDYT @rhuss ?

Reply:
I'm fine with either. However I suggest to merge this PR first, and then update #247 based on this.